sematic/versions.py

```diff
@@ -12,6 +13,23 @@
 # is made to the web API.
 MIN_CLIENT_SERVER_SUPPORTS = (0, 19, 0)
 
+# Support for dropping columns added in 3.35.0
+MIN_SQLITE_VERSION = (3, 35, 0)
+
+
+def _assert_sqlite_version():
+    version_tuple = sqlite3.sqlite_version.split(".")
+
+    # get major/minor as ints. Patch can sometimes have non-digit chars
+    major, minor = int(version_tuple[0]), int(version_tuple[1])
+    if (major, minor) < MIN_SQLITE_VERSION:
+        raise RuntimeError(
+            f"Sematic requires sqlite3 version to be at least "
+            f"{version_as_string(MIN_SQLITE_VERSION)}, but your python is using "
```

Comment:
Please format `3.35000.0` as `3.35.0`

Reply:
It should be. `3.35000.0` only ever showed up because I tested by increasing the minor version to something ridiculous to force a failure. The committed code has `3.35.0` though